### PR TITLE
Add media type registration

### PR DESCRIPTION
**Pull request by tfpauly**
Satisfied #9

### CODE REVIEW

File: draft-ietf-capport-api.md [hunk 1]
```diff
@@ -127,7 +127,42 @@ Information passed in this protocol may include a user's personal information, s
 
 # IANA Considerations
 
-TBD: None
+This document registers the media type for Captive Portal API JSON text, "application/captive+json".
+
+Type name: application
```
Comment (martinthomson) [inline draft-ietf-capport-api.md]: You might have more luck with formatting like:

```
label:

: value

label:

: value
```

I find that to be the best way to do these lists.

Comment (tfpauly) [reply draft-ietf-capport-api.md]: Ah, thanks! Updated.

File: draft-ietf-capport-api.md [hunk 2]
```diff
@@ -78,7 +78,7 @@ For example, if the Captive Portal API server is hosted at example.org, the URI'
 
 ## JSON Keys
 
-The Captive Portal API data structures are specified in JavaScript Object Notation (JSON) [RFC7159].
+The Captive Portal API data structures are specified in JavaScript Object Notation (JSON) [RFC7159]. Requests and responses for Captive Portal API SHOULD use the "application/captive+json" media type.
```
Comment (martinthomson) [inline draft-ietf-capport-api.md]: s/SHOULD// - no need to use the wishy-washy-est 2119 language here

Comment (darshakthakore) [reply draft-ietf-capport-api.md]: the sentence is reading a bit "incomplete" without a shall or should. Also, does it make sense to use a shall/must for the server. i.e a server sending this must set the content type and send this content type.

File: draft-ietf-capport-api.md [hunk 3]
```diff
@@ -78,7 +78,7 @@ For example, if the Captive Portal API server is hosted at example.org, the URI'
 
 ## JSON Keys
 
-The Captive Portal API data structures are specified in JavaScript Object Notation (JSON) [RFC7159].
+The Captive Portal API data structures are specified in JavaScript Object Notation (JSON) [RFC7159]. Requests and responses for the Captive Portal API use the "application/captive+json" media type.
```
Comment (darshakthakore) [inline draft-ietf-capport-api.md]: Maybe we can say..
Clients making a request to the Captive Portal API server can use the "application/captive+json" media type and Servers responding with the data structure specified below shall/must (?) include this in the Content-Type header of the response.

File: draft-ietf-capport-api.md (hunk 2 above)
Comment (tfpauly) [reply draft-ietf-capport-api.md]: Okay, added a sentence to clarify the client SHOULD and server MUST.

**Review by darshakthakore — APPROVED** [review]
I am ok with the added text.

**Pull request merged**